platform/src/main/java/org/stellar/anchor/platform/controller/sep/PlatformController.java

```diff
@@ -45,7 +45,16 @@ public PatchTransactionsResponse patchTransactions(@RequestBody PatchTransaction
   @RequestMapping(
       value = "/transactions",
       method = {RequestMethod.GET})
-  public GetTransactionResponse getTransactions() throws AnchorException {
-    throw new NotFoundException("Not implemented");
+  public GetTransactionsResponse getTransactions(
+      @RequestParam(value = "sep") TransactionsSeps sep,
+      @RequestParam(required = false, value = "order_by", defaultValue = "created_at")
+          TransactionsOrderBy order_by,
+      @RequestParam(required = false, value = "order", defaultValue = "asc") Sort.Direction order,
+      @RequestParam(required = false, value = "statuses") List<SepTransactionStatus> statuses,
```

Comment:
Can you think of a concrete use case where the business would need to get transactions in multiple statuses? At that point, why not just get all transactions and filter on the client side?

Reply:
Statuses is optional. The behavior you describing can be achieved with not providing statuses at all.
The business may want to filter orders in multiple statuses to expire them, for example. 
If they want to expire both pending_user_transfer_start and incomplete after certain amount without any action